bin/blanksky

```diff
@@ -207,12 +206,119 @@ def check_tool_par(evt,tool,param):
+    # 92016001.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis2D2000-12-01bgstow_ctiN0003.fits
+    # 92016001.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis3D2000-12-01bgstow_ctiN0005.fits
+    # 92016001.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis5D2000-12-01bgstow_ctiN0002.fits
+    # 92016001.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis6D2000-12-01bgstow_ctiN0005.fits
+    # 92016001.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis7D2000-12-01bgstow_ctiN0002.fits
+    # 92016001.0
+    ## non-CTI corrected files ##
+    # $CALDB/data/chandra/acis/bkgrnd/acis5D2000-12-01bgstowN0004.fits
+    # 92016001.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis7D2000-12-01bgstowN0004.fits
+    # 92016001.0
+
+    ### Group E ###
+    # $CALDB/data/chandra/acis/bkgrnd/acis0D2005-09-01bgstow_ctiN0002.fits
+    # 241920064.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis1D2005-09-01bgstow_ctiN0002.fits
+    # 241920064.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis2D2005-09-01bgstow_ctiN0002.fits
+    # 241920064.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis3D2005-09-01bgstow_ctiN0002.fits
+    # 241920064.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis5D2005-09-01bgstow_ctiN0002.fits
+    # 241920064.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis6D2005-09-01bgstow_ctiN0002.fits
+    # 241920064.0
+    # $CALDB/data/chandra/acis/bkgrnd/acis7D2005-09-01bgstow_ctiN0002.fits
+    # 241920064.0
+
+    ### Group F ###
+    # $CALDB/data/chandra/acis/bkgrnd/acis0D2009-09-21bgstow_ctiN0002.fits
+    # 369878466.184
+    # $CALDB/data/chandra/acis/bkgrnd/acis1D2009-09-21bgstow_ctiN0002.fits
+    # 369878466.184
+    # $CALDB/data/chandra/acis/bkgrnd/acis2D2009-09-21bgstow_ctiN0002.fits
+    # 369878466.184
+    # $CALDB/data/chandra/acis/bkgrnd/acis3D2009-09-21bgstow_ctiN0002.fits
+    # 369878466.184
+    # $CALDB/data/chandra/acis/bkgrnd/acis5D2009-09-21bgstow_ctiN0002.fits
+    # 369878466.184
+    # $CALDB/data/chandra/acis/bkgrnd/acis6D2009-09-21bgstow_ctiN0002.fits
+    # 369878466.184
+    # $CALDB/data/chandra/acis/bkgrnd/acis7D2009-09-21bgstow_ctiN0002.fits
+    # 369878466.184
+    """
+
+    # Assumes data sets are CTI-corrected, ignore the non-CTI-corrected
+    # Epoch D set of early ACIS-5 and -7 uncorrected stowed files.
+    # This will almost certainly affect all graded-mode observations, but
+    # if the observation's made in this mode, it's likely so bright that
+    # the user wouldn't be using blanksky/particle backgrounds to being with
+    try:
+        ctiapp = fileio.get_keys_from_file(f"{evtfile}[#row=0]")["CTI_APP"]
+    except KeyError:
+        raise IOError(f"{evtfile} is missing the 'CTI_APP' header keyword!")
+
+    if "N" in "".join(set(ctiapp)).upper():
+        raise IOError(f"The '{toolname}' script only supports CTI-corrected event files for ACIS stowed background file selection.")
+    
+    caldb = os.environ["CALDB"]
+
+    bkg_path = f"{caldb}/data/chandra/acis/bkgrnd"
+
+    epoch_dict = {"Epoch D" : {"tstart" : 92016001.0,
```

Comment:
As an aside, I wonder if we can just query the CALDB to find the necessary data (but then it might be hard to do fall back/forward cases).

Reply:
One would hope, but I checked.  There's no unique keyword or product query type  that's unique to the stowed background, they're indistinguishable from regular blanksky files, and the caldb4 (BKGRND) query returns only blanksky files.

Reply:
Joy...